code/modules/modular_computers/file_system/programs/phys_scanner.dm

```diff
@@ -0,0 +1,65 @@
+#define TABLET_SCANNER_NONE "None"
+#define TABLET_SCANNER_MEDICAL "Medical"
+#define TABLET_SCANNER_REAGENT "Reagent"
```

Comment:
If these aren't needed outside of this file, I'd recommend undefining them at the end of the file.